monitoring/update-bucket-capacity-info-cronjob/alerts.yaml

```diff
@@ -0,0 +1,99 @@
+                kube_job_owner{owner_name="${update_bucket_capacity_info_cronjob}"}
+              )
+              BY (job_name, owner_name, namespace)
+              == ON(owner_name) GROUP_LEFT()
+              max(
+                kube_job_status_start_time
+                * ON(job_name,namespace) GROUP_RIGHT()
+                kube_job_owner{owner_name="${update_bucket_capacity_info_cronjob}"}
+              )
+              BY (owner_name),
+              "cronjob", "$1", "owner_name", "(.*)")
+          ) > ${update_bucket_capacity_info_job_duration_threshold}
+        labels:
+          severity: critical
+        annotations:
+          description: "Job {{$labels.job_name}} is taking more than 4m to complete"
+          summary: "Job {{$labels.job_name}} didn't finish after 4m"
+
+      - alert: UpdateBucketCapacityJobFailed
+        expr: |    
+          clamp_max(
+            label_replace(
+              max(
+                kube_job_status_start_time
+                * ON(job_name,namespace) GROUP_RIGHT()
+                kube_job_owner{owner_name="${update_bucket_capacity_info_cronjob}"}
+              )
+              BY (job_name, owner_name, namespace)
+              == ON(owner_name) GROUP_LEFT()
+              max(
+                kube_job_status_start_time
+                * ON(job_name,namespace) GROUP_RIGHT()
+                kube_job_owner{owner_name="${update_bucket_capacity_info_cronjob}"}
+              )
+              BY (owner_name),
+              "cronjob", "$1", "owner_name", "(.+)")
+            , 1)
+          * ON(job_name) GROUP_LEFT()
+          (kube_job_status_failed > 0)
+        labels:
+          severity: warning
+        annotations:
+          description: "Job {{$labels.job_name}} failed to complete"
+          summary: "Job {{$labels.job_name}} failed"
+
+      - alert: NoSuccessfulUpdateBucketCapacityJobRunIn10m
+        expr: |
+          (
+            time() -
+            label_replace(
+              max(
+                kube_job_status_completion_time
+                * ON(job_name,namespace) GROUP_RIGHT()
+                kube_job_owner{owner_name="${update_bucket_capacity_info_cronjob}"}
+              )
+              BY (job_name, owner_name, namespace)
+              == ON(owner_name) GROUP_LEFT()
+              max(
+                kube_job_status_completion_time
+                * ON(job_name,namespace) GROUP_RIGHT()
+                kube_job_owner{owner_name="${update_bucket_capacity_info_cronjob}"}
+              )
+              BY (owner_name),
+              "cronjob", "$1", "owner_name", "(.+)")
+          ) > ${update_bucket_capacity_info_success_job_existence_duration_threshold}
+        labels:
+          severity: critical
+        annotations:
+          description: |
+            No successful {{$labels.cronjob}}
+            job run in the last 10m, will cause bucket capacity to be out of date
+            and Veeam to fail to process.
```

Comment:
nit: I feel this formulation to be a bit too vague. Maybe we can be a bit more precise on Veeam impacts ?